Packs/ApiModules/ReleaseNotes/2_2_21.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CoreIRApiModule
+Improved implementation by consolidating the shared command **get-incidents** between **CortexCoreIR** and **CortexXDRIR** into the shared module.
```

Comment:
Users are not exposed to CoreIRApiModule and I think this comment is added to all packs that uses the API module
```suggestion
- Improved implementation of *xdr-get-incidents* by unifying similar code blocks.
```